src/Chr.Avro/Abstract/SchemaBuilder.cs

```diff
@@ -100,15 +105,20 @@ public class SchemaBuilder : ISchemaBuilder
         /// A resolver to retrieve type information from. If no resolver is provided, the schema
         /// builder will use the default <see cref="DataContractResolver" />.
         /// </param>
-        public SchemaBuilder(IEnumerable<Func<ISchemaBuilder, ISchemaBuilderCase>> caseBuilders, ITypeResolver typeResolver = null)
+        /// <param name="temporalBehavior">
+        /// The behavior of how temporal types like DateTime are being serialized.
+        /// Options include Iso8601 (string based), EpochMicroseconds and EpochMilliseconds (long based)
+        /// </param>
+        public SchemaBuilder(IEnumerable<Func<ISchemaBuilder, ISchemaBuilderCase>> caseBuilders, ITypeResolver typeResolver = null,
```

Comment:
remove the `temporalBehavior` parameter from this constructor (since it’s an argument for the case builders, not for the schema builder itself) and remove the `?? CreateCaseBuilders(...)` from the body

that will align with how we do it on `ReflectionResolver`

Reply:
Like this? 
https://github.com/promontis/dotnet-avro/commit/f7629c5cb6f5218dbf24b5c0b932610c7239acee

Reply:
Note line 92